packages/typespec-ts/test/integration/generated/parameters/collection-format/src/index.d.ts

```diff
@@ -7,13 +7,31 @@ import { StreamableMethod } from '@azure-rest/core-client';
 
 export declare function buildCsvCollection(items: string[] | number[]): string;
 
+export declare function buildExplodedFormStyle<ValueType>(value: ValueType): {
+    explode: true;
+    style: "form";
+    value: ValueType;
+};
+
 export declare function buildMultiCollection(items: string[], parameterName: string): string;
 
-export declare function buildPipeCollection(items: string[] | number[]): string;
+export declare function buildUnexplodedFormStyle<ValueType>(value: ValueType): {
```

Comment:
is it possible to pass the explode and style as parameter in the helper function and build one helper function? I find it quite confusing if there's multiple different helpers. Beside I think typespec/openapi docs do have those information, customers are not stranger to those if they are authoring RLC code from the typespec/openapi references ?